library/elasticsearch

```diff
@@ -1,21 +1,25 @@
-# this file is generated via https://github.com/docker-library/elasticsearch/blob/47a72b6f1962df04a62616b1c3c32e85b5410639/generate-stackbrew-library.sh
+# this file is generated via https://github.com/docker-library/elasticsearch/blob/d22dc8b36939ab325763fa92e4e6ee3059aaea59/generate-stackbrew-library.sh
 
 Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
              Joseph Ferguson <yosifkit@gmail.com> (@yosifkit)
```

Comment:
We'll probably want to update this field at some point. :smile: :heart:

(IMO, not worth holding this up for that.)